CheatSheets/windows-cheatsheet.md

```diff
@@ -146,3 +147,21 @@
 | `Backspace` | Go back to the settings home page |
 
 **[🔼Back to Top](#table-of-contents)**
+
+## Touchpad Shortcuts
+
+| Gesture | Description |
+| ------- | ----------- |
+| `Single Tap` | Single click |
+| `Tap with two fingers` | Right click |
+| `Tap twice and drag` | Multi Select |
+| `Drag two fingers` | Scroll |
+| `Pinch` | Zoom |
```

Comment:
```suggestion
| `Pinch` 🤏| Zoom |
```

Reply:
If you add a pinching hand, then it will be better to realize.

Reply:
done